README.md

```diff
@@ -129,14 +140,13 @@ Example are available, see below:
 
 ```sh
 # Install a qserv instance with custom settings
-kubectl apply -k overlays/ncsa_dev/ --namespace='qserv-dev'
-
-# Install a qserv instance with custom settings
-kubectl apply -k overlays/ncsa_prod/ --namespace='qserv-prod'
+kubectl apply -k https://github.com/lsst/qserv-operator/overlays/ncsa_dev --namespace='qserv-prod'
 ```
 
-In order to create a customized Qserv instance, create a `Kustomize` overlay using instruction below:
+In order to create a customized Qserv instance, create a `Kustomize` overlay using instructions below:
 ```sh
+git clone https://github.com/lsst/qserv-operator.git
+cd qserv-operator
 cp -r overlays/local/ overlays/my-qserv/
```

Comment:
my-qserv is "pick your name, right? sometimes it's more clear to put that name in angle brackets, like `overlays/<kustomization name>`. With this operator stuff (k8s in general, really) it can be really hard to tell when a name is meaningful and/or has to match another name someplace, and when it's just arbitrary. Whatever we can do to help make that easier for readers would be 💯

Reply:
:+1: